src/views/ActivityAppFeed.vue

```diff
@@ -208,12 +208,10 @@ watch(props, () => {
 	&__container {
 		display: flex;
 		flex-direction: column;
-
 		height: 100%;
-		width: min(100%, 924px);
-		max-width: 924px;
-		margin: 0 auto;
-		padding-inline: 12px;
+		// Align with app navigation toggle
+		margin: var(--app-navigation-padding, 8px) 0 0 calc(2 * var(--app-navigation-padding, 8px) + 44px);
+		padding-inline: 0 44px;
```

Comment:
If this padding is defined by the a button size
```suggestion
		padding-inline: 0 var(--default-clickable-area);
```